src/Features/CSharp/Portable/Completion/KeywordRecommenders/StackAllocKeywordRecommender.cs

```diff
@@ -16,20 +16,48 @@ public StackAllocKeywordRecommender()
 
         protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
         {
-            // type t = |
-            var token = context.TargetToken;
-            if (token.IsUnsafeContext())
+            var node = context.TargetToken.Parent;
+
+            // When in an empty string
```

Comment:
Needs to eb changed.  this sounds like "if the user has ```var v = "$$"```.  We want to say "start of file" as "empty string" is really ambiguous.